### PR TITLE
Convert to Python3

### PR DESCRIPTION
**Pull request by ralic**
using : find . -name '*.py' | xargs 2to3 -w

### CODE REVIEW

File: akamai/edgegrid/edgegrid.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
     from urllib.parse import urlparse, parse_qsl, urlunparse
 else:
```
Comment (elreydetodo) [inline akamai/edgegrid/edgegrid.py]: This bit of the code will need to be reworked. There's no reason to leave it this way if we're removing Python 2 support.

File: akamai/edgegrid/edgegrid.py.bak [hunk 2]
```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
```
Comment (elreydetodo) [inline akamai/edgegrid/edgegrid.py.bak]: This whole file is unnecessary. We're using git.

**Comment by elreydetodo** [comment]
I'm not sure this is a good idea until macOS upgrades their base Python version to something in the 3 series. There is very little that needs to be changed, and it's mostly isolated with major version checks already.

**Comment by justinrummel** [comment]
Python 2 support will be dropped on the next release (Catalina), however, if we follow the Apple n-2 we need to have both Python2 and Python3 support for a couple more years just in case older versions of the OS are being used.  Not everyone can update the following week of an OS release. :)

**Pull request closed without merging**